spec/index.emu

```diff
@@ -74,6 +74,7 @@ location: https://rawgit.com/tc39/proposal-realms/master/index.html
         1. Set _O_.[[Realm]] to _realmRec_.
         1. Perform ? SetRealmGlobalObject(_realmRec_, *undefined*, *undefined*).
         1. Perform ? SetDefaultGlobalBindings(_O_.[[Realm]]).
+        1. Perform ? HostInitializeUserRealm(_O_.[[Realm]]).
```

Comment:
In [`InitializeHostDefinedRealm`](https://tc39.es/ecma262/#sec-initializehostdefinedrealm), we do: 

> `Create any implementation-defined global object properties on globalObj.` 

I wonder if that's something that we should do here as part of `HostInitializeUserRealm`, or doing it here.